src/wallet/init.h

```diff
@@ -8,18 +8,36 @@
 
 #include <string>
 
+class CRPCTable;
+class CScheduler;
+
 //! Return the wallets help message.
 std::string GetWalletHelpString(bool showDebug);
 
 //! Wallets parameter interaction
 bool WalletParameterInteraction();
 
+//! Register wallet RPCs.
+void RegisterWalletRPC(CRPCTable &tableRPC);
+
 //! Responsible for reading and validating the -wallet arguments and verifying the wallet database.
 //  This function will perform salvage on the wallet if requested, as long as only one wallet is
 //  being loaded (CWallet::ParameterInteraction forbids -salvagewallet, -zapwallettxes or -upgradewallet with multiwallet).
```

Comment:
Comment should say "WalletParameterInteraction"

Reply:
added as trivial commit